### DISCUSSION TITLE
☝️ Moving support from Discord to Github Discussions

### DISCUSSION THREAD

**Discussion by clemcode** [nuxt/nuxt#9285] [Questions (Nuxt 2)]
Over the years, Nuxt's Discord server has become the best place to get help, thanks to a vibrant community of users and moderators. But as the Nuxt user base increases, the "Help" channels start to show some limitations. We think that [Github Discussions](https://docs.github.com/en/discussions) would be more useful and manageable for the community.

## Discord limitations

- As there are lots of questions, if no answer is provided in a few hours, the lonely question gets flooded by the new ones. This is of course frustrating for the user asking for help.
- At the same time, answering all of the questions is very time consuming.
- The value provided by the answers is time-limited, as they can't be archived, searched, and referenced later, and are not indexed by search engines.

## Github Discussions

After consulting the Nuxt framework team and key contributors and moderators, we came to the conclusion that Github Discussions seem to check all requirements :

- Public
- Search, Categorize, Upvote answers
- A Ranking system for active contributors which shows more recognition for the most helpful community members.

A [Q&A channel](https://github.com/nuxt/nuxt.js/discussions/categories/q-a) has already been created, and we want to increase its usage in the coming weeks.

## What's coming next ?

Discord channels won't be closed, as we feel they are still useful for broad topics (Should I use X, How do you see the future of pattern Y...) and one-liners, but every help request and specific questions would find a better place in a Github Discussion.

An introduction to Github discussions, contribution guidelines and Code of Conduct will be edited and pinned [here](https://github.com/nuxt/nuxt.js/discussions) to guide everyone in the transition.

We're confident this transition will increase the quality of support. Feel free to ask any questions or give us feedback below! 👇

**Comment by kissu** [comment]
Perfect to see that Nuxt is migrating to here, I disliked Discord for those exact reasons ! Github's discussions are so neat! ❤️ 
I'm currently trying to help people on Stackoverflow with the [[nuxt.js] tag](https://stackoverflow.com/questions/tagged/nuxt.js?sort=Newest&uqlId=36985).

I was wondering if you had any opinions on the relation between the 2:
- do you think some questions are more appropriate here or there?
- should the existence of those discussions impact SO anyhow?

EDIT: I mean, Discussions are not for help only, it's a first point and the UI is not as bad so it's already a 2 - 0. 😆
